smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java

```diff
@@ -56,45 +55,91 @@
 
     private final MultiUserChatManager mucManagerOne;
     private final MultiUserChatManager mucManagerTwo;
+    private final MultiUserChatManager mucManagerThree;
     private final DomainBareJid mucService;
 
     public MultiUserChatIntegrationTest(SmackIntegrationTestEnvironment environment)
-                    throws NoResponseException, XMPPErrorException, NotConnectedException,
-                    InterruptedException, TestNotPossibleException {
+            throws NoResponseException, XMPPErrorException, NotConnectedException,
+            InterruptedException, TestNotPossibleException {
```

Comment:
This appears to be an unrelated whitespace change. If so, please revert that.

Reply:
Done